piq/functional/layers.py

```diff
@@ -16,7 +17,7 @@ def __init__(self, kernel_size: int = 3, stride: int = 2, padding=1) -> None:
         self.stride = stride
         self.padding = padding
 
-        self.kernel = None
+        self.kernel: Optional[torch.Tensor] = None
```

Comment:
Here I think it is good to have type annotation here because it shows target data type